build.gradle

```diff
@@ -54,32 +54,29 @@ repositories {
     }
 }
 
-
 dependencies {
     api platform('com.exactpro.th2:bom:3.2.0')
 
-    implementation 'com.exactpro.th2:grpc-common:3.11.1'
     implementation 'com.exactpro.th2:common:3.40.0'
+    implementation 'com.exactpro.th2:grpc-codec:0.0.2'
 
     implementation 'io.github.microutils:kotlin-logging:2.1.23'
 
-    implementation 'org.slf4j:slf4j-api'
-    implementation 'org.slf4j:slf4j-log4j12'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
+    implementation 'org.apache.logging.log4j:log4j-1.2-api'
```

Comment:
We don't need this in our implementation dependencies. Only logging API should be in implementation dependencies. They all are provided by common library